packages/fuel-gauge/src/e2e-script.test.ts

```diff
@@ -47,8 +67,7 @@ describe('Live Script Test', () => {
       console.error((e as Error).message);
       console.warn(`
         not enough coins to fit the target?
-        - add assets: https://faucet-testnet.fuel.network/
-        - check balance: https://app.fuel.network/account/${address}/assets/
+        - add assets: https://faucet-${selectedNetwork}.fuel.network/
```

Comment:
Did you consider having a `faucetUrl` property for [network members](https://github.com/FuelLabs/fuels-ts/blob/89667c895825a4d3ee9c98f928d8b517dcfc66c1/packages/fuel-gauge/src/e2e-script.test.ts#L12-L21)?